Basic-Car-Maintenance/Shared/MainView/Views/MainTabView.swift

```diff
@@ -40,6 +41,16 @@ struct MainTabView: View {
                     Label("Settings", systemImage: "gear")
                 }
         }
+        .sheet(isPresented: $isShowingAlert) {
+            
+        } content: {
+            AlertView()
+                .presentationDetents([.medium])
+        }
```

Comment:
```suggestion
        .sheet(isPresented: $isShowingAlert) {
            AlertView()
                .presentationDetents([.medium])
        }
```

You can change this to just not include the `onDismiss` parameter if you're not going to use it